commands/user.go

```diff
@@ -241,6 +241,7 @@ func init() {
 	UserCreateCmd.Flags().String("locale", "", "Optional. The locale (ex: en, fr) for the new user account")
 	UserCreateCmd.Flags().Bool("system_admin", false, "Optional. If supplied, the new user will be a system administrator. Defaults to false")
 	UserCreateCmd.Flags().Bool("email_verified", false, "Optional. If supplied, the new user will have the email verified. Defaults to false")
+	UserCreateCmd.Flags().Bool("disable-welcome-email", false, "Optional. If supplied, the new user will not receive a welcome email. Defaults to false")
```

Comment:
Why is "disable-welcome-email" logic being added in this PR?

Reply:
I'm guessing that this was caused by merging the `enable_enterprise` branch into this one. Unfortunately I need that merge because without it, the enterprise code will not run.

Reply:
I'm guessing this is because my branch is based off a version of master which is more recent than `enable_enterprise`.